### PR TITLE
fucky flags pt 2

### PR DESCRIPTION
**Pull request by Intigracy**
lockedvars is now a global list instead of a copypasted nightmare

added holder to it

fixes secret panel flags reqs again, my last change didn't fix the fact it requires those flags to use it instead of just see it.

### CODE REVIEW

File: code/global.dm [hunk 1]
```diff
@@ -6,6 +6,9 @@ var/round_end_info = ""
 //List of ckeys that have de-adminned themselves during this round
 var/global/list/deadmins = list()
 
+//List of vars that require DEBUG on top of VAREDIT to be able to edit
+var/global/list/lockedvars = list("vars", "client", "holder")
```
Comment (PJB3005) [inline code/global.dm]: `global` does nothing here.

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -3238,44 +3260,17 @@
...
-				J.spawn_positions = -1
-				message_admins("[key_name_admin(usr)] has removed the cap on security officers.")
-			if("virus_custom")
-				if(virus2_make_custom(usr.client))
-					feedback_add_details("admin_secrets_fun_used", "V_C")
-					message_admins("[key_name_admin(usr)] has trigger a custom virus outbreak.", 1)
+		if (usr)
+			log_admin("[key_name(usr)] used secret [href_list["secretsadmin"]]")
+			if (ok)
+				to_chat(world, text("<B>A secret has been activated by []!</B>", usr.key))
```
Comment (clusterfack) [inline code/modules/admin/topic.dm]: world?

Comment (clusterfack) [inline code/modules/admin/topic.dm]: This should probably be removed

**Comment by Intigracy** [comment]
k

**Pull request merged**